test/integration/spec/connect.js

```diff
@@ -831,7 +831,7 @@ describe('connect', function() {
   // eslint-disable-next-line no-warning-comments
   // TODO: The following tests verifies bitrates using default codec, OPUS.
   // We should also verify other codecs like ISAC, PCMU and PCMA.
-  describe('called with EncodingParameters', () => {
+  describe('called with EncodingParameters (@unstable: VIDEO-9969)', () => {
     const minAudioBitrate = 6000;
```

Comment:
This should be marked unstable only for firefox, make sure `isFireFox` is defined
```suggestion
  describe(`called with EncodingParameters ${isFirefox ? ' - @unstable: VIDEO-9969' : ''}`, () => {
    const minAudioBitrate = 6000;
```